packages/strapi-plugin-upload/admin/src/components/InputMedia/IconUpload.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+const IconUpload = () => {
+  return (
+    <svg
+      width="105"
+      height="84"
+      xmlns="http://www.w3.org/2000/svg"
+      xmlnsXlink="http://www.w3.org/1999/xlink"
+      style={{ margin: 'auto' }}
```

Comment:
Do we really want to set this style attribute here and not set it with a styled-component compo?

Reply:
This component was in the strapi-admin package. Anyway, this inline style is useless here. Nice catch.